REQUIRE

```diff
@@ -1 +1 @@
-julia 0.4
+julia 0.5-
```

Comment:
In that case you'd need to keep `julia 0.4` in REQUIRE, otherwise any version tagged after this is merged will not be installable on Julia 0.4, with or without Compat.